cypress/integration/pages/testsForAllPages.js

```diff
@@ -323,110 +322,30 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
     });
 
     describe('Header Tests', () => {
-      const hasLocalisedName =
```

Comment:
Removed the `hasLocalisedName` as all of the services seem to have this key value defined.

Reply:
The `news`, `scotland`, `archive` service configs do not have a `serviceLocalizedName`.

Reply:
But our integration tests currently are not run across these three services so imo we could opt this assertion for now.

Reply:
When we add integration tests for Optimo articles, we will need this.

Reply:
Is it safer to leave these tests in Cypress until they have a replacement in Integration? I onlt wanted to delete tests which were duplicated in Integration tests. 

We could also leave all PS sevices in the articles E2Es, rather than removing them - happy to update https://github.com/bbc/simorgh/issues/6657 to reflect this.